appserver/tests/pom.xml

```diff
@@ -42,6 +42,13 @@
 -->
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd"> <modelVersion>4.0.0</modelVersion>
+    <dependencies>
+        <dependency>
+            <groupId>jakarta.platform</groupId>
+            <artifactId>jakarta.jakartaee-web-api</artifactId>
+            <scope>compile</scope>
+        </dependency>
+    </dependencies>
```

Comment:
This should be `provided` rather than `compile`.
It's also already defined for the relevant samples already.

```suggestion
```

Reply:
> This should be `provided` rather than `compile`. It's also already defined for the relevant samples already.

For sure, I think it was accidentaly changed by IDE